databricks/koalas/tests/test_dataframe_conversion.py

```diff
@@ -70,3 +71,76 @@ def test_to_html(self):
             """)
         got = self.strip_all_whitespace(self.kdf.to_html(max_rows=2))
         self.assert_eq(got, expected)
+
+    @staticmethod
+    def setup_location(directory):
+        """Helper function to set up a temporary directory within test folder."""
+        if not os.path.exists(directory):
+            os.makedirs(directory)
+
+    @staticmethod
+    def teardown_location(location1, location2, directory):
+        """Helper function to remove the temporary directory and it's contents."""
+        if os.path.isfile(location1):
+            os.remove(location1)
+        if os.path.isfile(location2):
+            os.remove(location2)
+        if os.path.exists(directory):
+            os.rmdir(directory)
```

Comment:
We can use `TestUtils.temp_dir` or `TestUtils.temp_file`.

```py
class DataFrameConversionTest(ReusedSQLTestCase, SQLTestUtils, TestUtils):

    ...

    def test_xxx(self):
        with self.temp_dir() as tmp:
            ...
```

Reply:
Working on it.

Reply:
@ueshin, I have changed it according to your suggestions.
Please have a look.